csharp/test/Ice/ami/TestI.cs

```diff
@@ -77,17 +77,14 @@ public override bool
                 }
             }
 
-            public override void close(Test.CloseMode mode, Ice.Current current)
+            public override void close(Ice.Current current)
             {
-                switch (mode)
-                {
-                    case Test.CloseMode.Forcefully:
-                        abortConnection(current);
-                        break;
-                    default:
-                        closeConnection(current);
-                        break;
-                }
+                closeConnection(current);
```

Comment:
closeConnection and abortConnection below are/were placeholder for when the Slice operations are renamed.

We should not keep them all!

Reply:
Alright, I see now.
I inlined these extra functions.